document/core/exec/modules.rst

```diff
@@ -613,10 +613,6 @@ It is up to the :ref:`embedder <embedder>` to define how such conditions are rep
 
     h. Let :math:`\X{eend}_i` be :math:`\X{eo}_i` plus the length of :math:`\elem_i.\EINIT`.
```

Comment:
Nit: Maybe move this down to where it is actually used first now.

Reply:
done